### DISCUSSION TITLE
How to shorten the display time for ui.notify

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#1383] [Q&A] [answered]
### Question

Hi All,

I am using a ```ui.notify``` to show a validation message on a login page. Like this:
```ui.notify(f'Wrong password, please try again', position='center', type='negative')```

The challenge now is after it comes up, it stays on the page for 2-3 seconds, of course the user can still keep working on the page and try again, but to me, it looks a bit odd. Is there a way to shorten the time for the display of the ```ui.notify```?

Initially I thought I will use the ```timer``` to make it go away after 1 sec, but I realized there is no ```visibility``` or that kind of property which I can use together with ```ui.notify```. Can you please guide me how best to do this?

Thanks a lot,
Anindya

**Comment by pythonormen** [comment, marked as answer]
If the message is the same it gathers in a summary / +1 notification so shouldn't cover the page. 

Not mentioned on the site but I think you can use the timeout prop to set a timer. See quasars site. 

Like

```py
ui.notify('Wrong password, please try again', timeout=1000, position='center', type='negative')
```

**Reply by Anindya088** [reply]
Hi @pythonormen ,

Thanks a ton, the ```timeout=1000``` did the trick 😄 

Also thanks for explaining how to use the Quasar elements. I was thinking I need to use ```props``` like with the other usual elements. But this one, now I understood that I can use inside main ```ui.notify``` as arguments. 

Thanks a lot,
Anindya
